tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -2841,7 +2841,7 @@ public function readFileInServerRootForOCIS(string $path):string {
 		$pathToOcis = \getenv("PATH_TO_OCIS");
 		$targetFile = \rtrim($pathToOcis, "/") . "/" . "services/web/assets" . "/" . ltrim($path, '/');
 		if (!\file_exists($targetFile)) {
-			throw new Exception('Target File' . $targetFile . 'could not be found');
+			throw new Exception('Target File' . $targetFile . ' could not be found');
```

Comment:
```suggestion
			throw new Exception('Target File ' . $targetFile . ' could not be found');
```
Meybe we need a space here too?